recipes/ignition-cmake/all/test_package/conanfile.py

```diff
@@ -5,7 +5,10 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def requirements(self):
+        self.requires("ignition-cmake/2.10.0@")
```

Comment:
`test_package` already adds implicit requirement. you may use `test_type = "explicit"` to avoid the error.